pkg/datastore/metadata.go

```diff
@@ -4,24 +4,35 @@ import (
 	"fmt"
 )
 
-// OperatorMetadata encapsulates operator metadata and manifest assocated with a package
+// OperatorMetadata encapsulates operator metadata and manifest
```

Comment:
This and other formatting fixes should have been done in the `[appregistry] move operator manifest type(s) out of appregistry` but I won't insist on it.